portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormRules.testcase

```diff
@@ -1213,12 +1348,10 @@ definition {
 		FormRules.viewNoRules();
 	}
 
-	@description = "This is a use case for LPS-119818."
+	@description = "LPS-80771 - Verify if an alert message is shown when the user tries to change the type of field that is being used in a Calculator Expression"
+	@ignore = "true"
 	@priority = "5"
-	test DeleteFieldThatBelongsToARule {
-		property portal.acceptance = "true";
-		property portal.upstream = "true";
-
+	test DisplaysAWarningWhenChangingAFieldUsedInACalculatorExpression {
```

Comment:
@leticiachiu, let's remove some letters `A` and name this test as `DisplaysWarningWhenChangingFieldUsedInCalculatorExpression`. This should be enough. :slightly_smiling_face: I'm suggesting it because the test name is bigger than the others. :wink: